policy-packs/aws-ts-finops/PulumiPolicy.yaml

```
@@ -0,0 +1,2 @@
+description: A collection of finops policies for AWS
```

```suggestion
description: A collection of FinOps policies for AWS
```

Pedantic, however, an homage to the Pulumi team member that coined the term!

---

I updated it. It should have been FinOps from the beginning.